### ISSUE TITLE
Impacted by Shutdown of Integration

### ISSUE THREAD

**Issue by LucasForster** [mheap/github-action-pr-heroku-review-app#17]
Just wanted to note that this action also seems to be impacted by the [current shutdown of the GitHub integration](https://status.heroku.com/incidents/2413).

I got these logs after running the action, but nothing shows up on my dashboard.

```
ℹ  info      User is a collaborator: admin
ℹ  info      PR opened by collaborator
☐  pending   Creating review app
☒  complete  Created review app
✔  success   Action complete
```

**Comment by Jonty800** [comment]
Hi @LucasForster are you able to confirm if this is still the case?

I'm looking for a solution that will allow me to create Review Apps in an automated way, just like before. Have you found anything that can replicate heroku's behaviour prior to the vulnerability, that works? :D Thanks!

**Comment by LucasForster** [comment]
I didn't try it since last week, but as far as I can tell nothing should have changed.

I'm in the exact same situation as you. As Heroku doesn't seem to act on restoring PR previews, I'm personally thinking of switching entirely to another provider, as this was my only justification for their high pricing. So yeah setting up a new CI, possibly using GitHub actions to avoid this whole issue in the future, might be my next move.

I didn't put in the necessary time in yet, but for AWS [mahbir/GitActionsDeploytoAWS](https://github.com/mahbir/GitActionsDeploytoAWS) and [this article](https://aws.amazon.com/blogs/devops/integrating-with-github-actions-ci-cd-pipeline-to-deploy-a-web-app-to-amazon-ec2/) seem to be a good starting point.

**Comment by Jonty800** [comment]
Yeah I agree. Bit of a nightmare. I'm thinking about setting up CI to trigger a build on Vercel when a PR is opened on [branch], and then deliver the build URL using a webhook with Slack. And that build can be deleted when the PR is closed

**Comment by ahmed2m** [comment]
I saw this on HN but haven't got to test it yet
https://gist.github.com/wyattades/7176227bd07b35782c05d1fe5000e282
